IntegrationWithPharmacies/Services/UrgentOrderService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using HealthClinic.CL.Adapters;
+using HealthClinic.CL.DbContextModel;
+using HealthClinic.CL.Dtos;
+using HealthClinic.CL.Model.Orders;
+using HealthClinic.CL.Repository;
+using IntegrationWithPharmacies.FileProtocol;
+
+namespace IntegrationWithPharmacies.Services
+{
+    public class UrgentOrderService
+    {
+        private HttpService HttpService { get; }
+        private MedicineAvailabilityTable MedicineAvailabilityTable { get; }
+        public UrgentMedicineOrderRepository UrgentMedicineOrderRepository { get; }
+        public IUrgentMedicineOrderRepository IUrgentMedicineOrderRepository { get; }
+        public UrgentOrderService(MyDbContext context)
+        {
+            HttpService = new HttpService();
+            MedicineAvailabilityTable = new MedicineAvailabilityTable();
+            UrgentMedicineOrderRepository = new UrgentMedicineOrderRepository(context);
+        }
+
+        public UrgentOrderService(IUrgentMedicineOrderRepository urgentMedicineOrderRepository)
+        {
+            IUrgentMedicineOrderRepository = urgentMedicineOrderRepository;
+        }
+
+        public UrgentMedicineOrder Create(UrgentMedicineOrderDto dto)
+        {
+            return UrgentMedicineOrderRepository.Create(UrgentMedicineOrderAdapter.UrgentMedicineOrderDtoUrgentMedicineOrder(dto));
+        }
+
+        public List<UrgentMedicineOrder> GetAll()
+        {
+            return UrgentMedicineOrderRepository.GetAll();
+        }
+        public List<UrgentMedicineOrder> GetAllForStub()
+        {
+            return IUrgentMedicineOrderRepository.GetAll();
+        }
+        public UrgentMedicineOrder createIUrgentMedicineOrder(UrgentMedicineOrderDto dto)
+        {
+            return UrgentMedicineOrderAdapter.UrgentMedicineOrderDtoUrgentMedicineOrder(dto);
+        }
+        public Boolean SendOrderHttp(UrgentMedicineOrder order)
+        {
+            String medicineOrder = CreateOrder(order);
+            try
+            {
+                HttpService.SendUrgentOrder(medicineOrder);
+                return true;
+            }
+            catch (Exception e) { return false; }

```

Comment:
Write this in form:
"try
            {
                HttpService.SendUrgentOrder(CreateOrder(order));
                return true;
            }
            catch (Exception e) { return false; }"